cypress/e2e/app_spec.cy.js

```diff
@@ -0,0 +1,56 @@
+describe('App component', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:3000/')
+    cy.intercept('GET', 'http://localhost:8080/api/v1/places', {
```

Comment:
Question for learning: This intercept is confirmed to actually be intercepting in Cypress? I had the impression that the `cy.visit` line should be after the `cy.intercept` but that could be incorrect.